### PR TITLE
Small updates to data processing page

### PR DESCRIPTION
**Pull request by emiliom**
Replacement for PR #362 that was closed.

- added text about environmental parameters in EK60, based on exchanges on Slack. Includes text that attempts to be consistent with the Simard EK60 manual from https://www.simrad.online/ek60/ref_english/default.htm?startat=/ek60/ref_english/xxx_para_environment.html
- fixed small mistake

### CODE REVIEW

**Comment by lsetiawan** [comment]
Thanks for the fixes @emiliom! 🙏🏽 😄

File: docs/source/process.rst [hunk 1]
```diff
@@ -125,14 +125,19 @@ critical in biological interpretation of ocean sonar data. They influence:
 
 By default, echopype uses the following for calibration:
 
-- EK60 and EK80: Environmental parameters saved with the raw data files.
+- EK60 and EK80: Environmental parameters saved with the raw data files. 
+  For EK60, instrument operators may enter temperature and salinity values into the 
+  `Simrad EK60 software's Environment dialog 
+  <https://www.simrad.online/ek60/ref_english/default.htm?startat=/ek60/ref_english/xxx_para_environment.html>`_
+  and the software will calculate sound speed, or directly enter sound speed instead.
+  Only sound speed is saved into the raw file.
```
Comment (leewujung) [inline docs/source/process.rst:133]: Could you add "sound absorption" along with sound speed? Both are saved in the data files also.

Comment (emiliom) [reply docs/source/process.rst]: Done

**Comment by leewujung** [comment]
I'll merge this now. Thanks @emiliom !

**Pull request merged**